lib/crypto/secp256k1/secp256k1.go

```diff
@@ -36,6 +37,16 @@ type PublicKey struct {
 	key ecdsa.PublicKey
 }
 
+// SignVerify verify signature given pubkey and msg
```

Comment:
nit fixing up the comment :wink: 

```suggestion
// SignVerify verifies a signature given a public key and a message
```